pkg/streamingpromql/functions.go

```diff
@@ -570,75 +570,76 @@ func SortOperatorFactory(descending bool) InstantVectorFunctionOperatorFactory {
-	"tanh":               InstantVectorTransformationFunctionOperatorFactory("tanh", functions.Tanh),
-	"timestamp":          TimestampFunctionOperatorFactory,
-	"vector":             scalarToInstantVectorOperatorFactory,
-	"year":               TimeTransformationFunctionOperatorFactory("year", functions.Year),
+	"abs":                          InstantVectorTransformationFunctionOperatorFactory("abs", functions.Abs),
+	"absent":                       AbsentFunctionOperatorFactory,
+	"absent_over_time":             AbsentOverTimeFunctionOperatorFactory,
+	"acos":                         InstantVectorTransformationFunctionOperatorFactory("acos", functions.Acos),
+	"acosh":                        InstantVectorTransformationFunctionOperatorFactory("acosh", functions.Acosh),
+	"asin":                         InstantVectorTransformationFunctionOperatorFactory("asin", functions.Asin),
+	"asinh":                        InstantVectorTransformationFunctionOperatorFactory("asinh", functions.Asinh),
+	"atan":                         InstantVectorTransformationFunctionOperatorFactory("atan", functions.Atan),
+	"atanh":                        InstantVectorTransformationFunctionOperatorFactory("atanh", functions.Atanh),
+	"avg_over_time":                FunctionOverRangeVectorOperatorFactory("avg_over_time", functions.AvgOverTime),
+	"ceil":                         InstantVectorTransformationFunctionOperatorFactory("ceil", functions.Ceil),
+	"changes":                      FunctionOverRangeVectorOperatorFactory("changes", functions.Changes),
+	"clamp":                        ClampFunctionOperatorFactory,
+	"clamp_max":                    ClampMinMaxFunctionOperatorFactory("clamp_max", false),
+	"clamp_min":                    ClampMinMaxFunctionOperatorFactory("clamp_min", true),
+	"cos":                          InstantVectorTransformationFunctionOperatorFactory("cos", functions.Cos),
+	"cosh":                         InstantVectorTransformationFunctionOperatorFactory("cosh", functions.Cosh),
+	"count_over_time":              FunctionOverRangeVectorOperatorFactory("count_over_time", functions.CountOverTime),
+	"day_of_month":                 TimeTransformationFunctionOperatorFactory("day_of_month", functions.DayOfMonth),
+	"day_of_week":                  TimeTransformationFunctionOperatorFactory("day_of_week", functions.DayOfWeek),
+	"day_of_year":                  TimeTransformationFunctionOperatorFactory("day_of_year", functions.DayOfYear),
+	"days_in_month":                TimeTransformationFunctionOperatorFactory("days_in_month", functions.DaysInMonth),
+	"deg":                          InstantVectorTransformationFunctionOperatorFactory("deg", functions.Deg),
+	"delta":                        FunctionOverRangeVectorOperatorFactory("delta", functions.Delta),
+	"deriv":                        FunctionOverRangeVectorOperatorFactory("deriv", functions.Deriv),
+	"double_exponential_smoothing": DoubleExponentialSmoothingFunctionOperatorFactory,
```

Comment:
(nit) perhaps a comment that the parser translates `holt_winters` into `double_exponential_smoothing` so there is no need to register it separately.

Reply:
2de690cdb2ebc607c43e651eb5a77c7b80b905bc